README.md

```diff
@@ -9,21 +9,370 @@ This repository contains the source code of the Ballerina Selenium library packa
 
 ## Overview
 
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+This module automates web applications across various browsers. Selenium interacts with web browsers directly, simulating user actions such as clicks, text input, page navigation, and more. 
 
-## Setup guide
+## Quickstart
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+### Import the module
 
-## Quickstart
+```ballerina
+import xlibb/selenium;
+```
+
+### Create a new web driver instance
+
+The core component of Selenium is the `WebDriver`. It is an interface for controlling a web browser instance and communicates with the browser through its native API.
+
+The constructor of the `WebDriver` takes `BrowserOptions` as an argument, which contains the following options:
+
+1. `browserName` - A enum value specifies the type of browser to open. Acceptable values are `Sikulix:CHROME` or `Sikulix:FIREFOX`. The default is `Sikulix:CHROME`.
+2. `url` - The URL of the web application to open in the browser.
+3. `headlessMode` - A boolean value indicating whether to run the browser in headless mode (without a GUI). The default is `false`.
+4. `incognitoMode` - A boolean value indicating whether to run the browser in incognito mode. The default is `false`.
+5. `additionalArguments` - A list of additional command-line arguments to pass to the browser during initialization.
+
+```ballerina
+// Opens a new Chrome browser instance and navigates to the specified URL.
+selenium:WebDriver driver = new ({
+    url: "https://central.ballerina.io/"
+});
+```
+
+### Locating elements
+
+`WebElement` in selenium represents an element in the DOM, which allows interaction (e.g., click, send keys).
```

Comment:
```suggestion
`WebElement` in Selenium represents an element in the DOM, which allows interaction (e.g., click, send keys).
```

Reply:
Let's start with a capitalized letter for all product names. Or you can use backticks with the lowercase word. (eg: `selenium`)